g2p/apply.py

```diff
@@ -56,20 +58,53 @@ def __init__(
         self.variants_number = variants_number
         self.word_list = word_list_file
         self.filter_empty_words = filter_empty_words
+        self.concurrent = concurrent
+
+        self.out_g2p_lexicon_parts = None
+        self.out_g2p_untranslated_parts = None
+        self.word_list_parts = None
```

Comment:
why do you define those as `None` first and then overwrite with the list later?
I think this can be deleted
```suggestion
```

Reply:
Yeah, I missed these when generalizing the concurrent=1 case, removed the redundant definitions now.